notes.md

```diff
@@ -3,27 +3,33 @@
 
 # Beginning of File
 
-An EBML file always starts with 0x1A. The 0x1A makes the DOS command "type" ends display. That way you can include ASCII text before the EBML data and it can be displayed. The EBML parser is safe from false-alarm with these ASCII only codes.
+An EBML file MUST start with `0x1A`. The 0x1A makes the DOS command "type" ends display. Therefore, ASCII text can be included before the EBML data and it can be displayed. After `0x1A`, the EBML header is stored. This allows the the parser to know what type of EBML file it is parsing.
 
-Next the EBML header is stored. This allows the the parser to know what type of EBML file it is parsing.
+# Mime Types
```

Comment:
What will the status of this section be? Will it stay as it is, or should we update it?

Reply:
As we're talking about [that mail extension thingy](https://en.wikipedia.org/wiki/MIME) and not about different types of actors, we should write it all-uppercase.

Reply:
Resolved (and conversation/question moved to) https://github.com/Matroska-Org/matroska-specification/pull/178